mu-plugins/blocks/notice/src/icon/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import { ReactComponent as Alert } from './library/alert.svg';
+import { ReactComponent as Info } from './library/info.svg';
+import { ReactComponent as Tip } from './library/tip.svg';
+import { ReactComponent as Warning } from './library/warning.svg';
+
+export default function ( { type /*, label */ } ) {
```

Comment:
```suggestion
export default function ( { type } ) {
```

Reply:
👍🏻  — I thought I'd go back and add an aria label, but changed approach.